spec/controllers/groups/questions/written/answered_controller_spec.rb

```diff
@@ -0,0 +1,69 @@
+require 'rails_helper'
+
+RSpec.describe Groups::Questions::Written::AnsweredController, vcr: true do
+  describe 'GET index' do
+    before(:each) do
+      get :index, params: { group_id: 'fpWTqVKh'}
+    end
+
+    it 'should have a response with http status ok (200)' do
+      expect(response).to have_http_status(:ok)
+    end
+
+    it 'assigns @answering_body, @answers and @answers_grouped_by_date' do
+      expect(assigns(:answering_body).type).to eq('https://id.parliament.uk/schema/AnsweringBody')
+
+      assigns(:answers).each do |answer|
+        expect(answer.type).to eq('https://id.parliament.uk/schema/Answer')
+      end
+
+      expect(assigns(:answers_grouped_by_date).keys.first).to be_a(Date)
+
+      expect(assigns(:answers_grouped_by_date).values.first.first.type).to eq('https://id.parliament.uk/schema/Answer')
+    end
+
+    it 'renders the index template' do
+      expect(response).to render_template('index')
+    end
+  end
+
+  describe '#data_check' do
```

Comment:
Is this test needed? `#data_check` is from `parliament-utils`? What additional functionality are you trying o test here?